ci/run_lint.sh

```diff
@@ -12,8 +12,17 @@ enter_suite 'lint'
 
 set -x
 
+csi_clean() {
+  rm "${BUILD_DIR}"/bin/test-includes-*
```

Comment:
Need `rm -f ...` in case no files match the glob, to avoid failing the script due to `set -e`.